Sources/SentrySwiftUI/Preview/SentryReplayMaskPreview.swift

```diff
@@ -3,7 +3,7 @@ import Sentry
 import SwiftUI
 import UIKit
 
-#if CARTHAGE || SWIFT_PACKAGE
+#if XCODE || SWIFT_PACKAGE
```

Comment:
`m`: But wait a second. We now always have the `DXCODE` swift flag set. This will always be true. So does this flag even make sense? We can also follow up with a PR.